### PR TITLE
codacy issues fixes

### PR DESCRIPTION
**Pull request by titusen**

### CODE REVIEW

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

File: src/zcash/util.cpp [hunk 1]
```diff
@@ -15,13 +15,12 @@ std::vector<unsigned char> convertIntToVectorLE(const uint64_t val_int) {
 // Convert bytes into boolean vector. (MSB to LSB)
 std::vector<bool> convertBytesVectorToVector(const std::vector<unsigned char>& bytes) {
     std::vector<bool> ret;
-    ret.resize(bytes.size() * 8);
+    ret.resize(bytes.size() * __CHAR_BIT__);
```
Comment (drgora) [inline src/zcash/util.cpp]: It seems that `__CHAR_BIT__` is not part of the c++ standard.
I would suggest using `CHAR_BIT`, which is guaranteed by the standard to be defined in `climits`.

Comment (titusen) [reply src/zcash/util.cpp]: done

**Review by drgora — APPROVED** [review]
Looks good to me, thanks

File: src/zcash/util.cpp [hunk 2]
```diff
@@ -15,13 +16,12 @@ std::vector<unsigned char> convertIntToVectorLE(const uint64_t val_int) {
...
     std::vector<bool> ret;
-    ret.resize(bytes.size() * 8);
+    ret.resize(bytes.size() * CHAR_BIT);
 
-    unsigned char c;
     for (size_t i = 0; i < bytes.size(); i++) {
-        c = bytes.at(i);
-        for (size_t j = 0; j < 8; j++) {
-            ret.at((i*8)+j) = (c >> (7-j)) & 1;
+        unsigned char c = bytes.at(i);
```
Comment (a-petrini) [inline src/zcash/util.cpp:22]: Why moving the declaration of the `c` variable inside the loop?

**Review by a-petrini — APPROVED** [review]
Looks good, I only left a minor question

File: src/zcash/util.cpp (hunk 2 above)
Comment (titusen) [reply src/zcash/util.cpp:22]: Cause it's not used outside the loop.

Comment (a-petrini) [reply src/zcash/util.cpp:22]: Yes, true. I am more concerned by the optimization matter, as we are re-declaring the variable at every iteration. I think that the compiler would optimize it, however.

Comment (a-petrini) [reply src/zcash/util.cpp:22]: After further investigation, I see there is no impact on the performance, even without compiler optimization.

**Pull request merged**